Podcast/InternalProfileViewController.swift

```diff
@@ -192,11 +193,21 @@ class InternalProfileViewController: ViewController, UITableViewDelegate, UITabl
             cell.selectionStyle = .gray
             return cell
         case subscriptionsTableView:
-            let cell = tableView.dequeueReusableCell(withIdentifier: reusableSubscriptionCellID, for: indexPath) as? SearchSeriesTableViewCell ?? SearchSeriesTableViewCell()
-            if let series = subscriptions?[indexPath.row] {
-                cell.configure(for: series, index: indexPath.row, showLastUpdatedText: true)
+            guard let subs = subscriptions else { return UITableViewCell() }
+            if subs.count > 0 {
+                let cell = tableView.dequeueReusableCell(withIdentifier: reusableSubscriptionCellID, for: indexPath) as? SearchSeriesTableViewCell ?? SearchSeriesTableViewCell()
+                if let series = subscriptions?[indexPath.row] {
+                    cell.configure(for: series, index: indexPath.row, showLastUpdatedText: true)
+                }
+                cell.backgroundColor = .offWhite
+                return cell
+            }
+            else {
+                guard let user = System.currentUser else { return UITableViewCell() }
+                let cell = NullProfileTableViewCell(user: user)
+                tableView.backgroundColor = .paleGrey
```

Comment:
why are we changing the background color here?

Reply:
![image](https://user-images.githubusercontent.com/34046832/37190069-bb541d5a-2325-11e8-82ac-90df85322064.png)
the default color is white for the SearchSeriesTableViewCells

Reply:
Oh if I just set it to paleGrey in viewDidLoad it doesn't show through the series cells

Reply:
Oh I can just change the backgroundColor in viewDidLoad, I wasn't sure if the .offwhite that it was set to would show through